mlrun/model_monitoring/stream_processing.py

```diff
@@ -478,8 +477,9 @@ def do(self, full_event):
 
         # Create a storey event object with list of events, based on endpoint_id which will be used
         # in the upcoming steps
-        storey_event = storey.Event(body=events, key=endpoint_id)
-        return storey_event
+        full_event.key = endpoint_id
```

Comment:
What will happen on the next step? IIRC it `storey.FlatMap`... all the new events will have the same id? Is it ok?

Reply:
The same key. The flatten with the same endpoint ID is the cause of [ML-9667](https://iguazio.atlassian.net/browse/ML-9667), but this PR does not aim to fix it.